tests/backend/specs/api/api.js

```diff
@@ -21,6 +23,7 @@ const testPadId = makeid();
 describe(__filename, function () {
   describe('API Versioning', function () {
     it('errors if can not connect', function (done) {
+      this.timeout(150);
```

Comment:
150ms is a really short timeout. Intentional? Ditto for the timeouts added below.

Reply:
Yea, it's intentional, these timeouts are due to catch changes that make processes/execution slower, they are as close to execution time(on a slow device) as possible.